projects/js-packages/ai-client/src/components/ai-control/index.tsx

```diff
@@ -100,19 +115,34 @@ export function AIControl(
 		'enter',
 		e => {
 			e.preventDefault();
-			onSend?.( value );
+			handleSend();
 		},
 		{
 			target: promptUserInputRef,
 		}
 	);
 
+	useKeyboardShortcut(
+		'esc',
+		e => {
+			e.stopImmediatePropagation();
```

Comment:
Why do we need to stop the event propagation here?

Reply:
To avoid double trigger in the AI Assistant because of this:
https://github.com/Automattic/jetpack/pull/33271#discussion_r1336206797